models/labels/balancer_v2/labels_balancer_v2_sources.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+
+sources:
+  - name: balancer_v2_arbitrum
+    tables:
+      - name: InvestmentPoolFactory_call_create
+      # TODO Add and define source tables here
+      #  columns:
```

Comment:
After source tables added, we can define the undefined(not defined before in the spellbook project) source tables in the `sources.yml` file, like we did for other balancer spells. https://github.com/duneanalytics/spellbook/blob/main/models/balancer/ethereum/balancer_ethereum_sources.yml

Reply:
I pushed a commit adding these source tables to the .yml file and another one updating the description of the sources that already existed on the balancer_v2 `sources.yml` file folders. @mendesfabio please review if all descriptions are correct.